### ISSUE TITLE
Speech to Text 

### ISSUE THREAD

**Issue by kevinlee12** [kevinlee12/CASCategorization#7]
This is for discussion purposes. Should we implement a speech to text journal entry box? (This will be limited to only text, not pics or vids yet)

Comments? Leave them below!

**Comment by nsivapalan** [comment]
@kevinlee12 definitely! perhaps after everything is finished though. Here is a good example: https://dictation.io/

**Comment by kevinlee12** [comment]
Google Speech API: https://cloud.google.com/speech/
